hack/scripts/prep-release.sh

```diff
@@ -40,6 +40,29 @@ update_version() {
 	go run ./hack/tools/generate-npm-packages
 }
 
+add_redirects() {
+	local VER="$1"
+	local STANZA=$(
+		cat <<EOF
+
+[[redirects]]
+from = "/cerbos/$VER/_images/*"
```

Comment:
Won't this add a redirect for the to-be-released version number (which will actually be "latest")? We need it for the previous version.

Although, it might be fine to add the redirect preemptively - we'd just need to manually add 0.34 in now since the next one to be added will be 0.35

Reply:
Yeah, the idea was to add the entry preemptively because the script doesn't have an easy way of determining what the soon-to-be-outdated version is.

I forgot to add 0.34 because in my mind we were still on 0.33 🤦🏽 Thanks for catching that.